engine/dlib/src/dlib/file_descriptor_posix.cpp

```diff
@@ -12,16 +12,17 @@
 // CONDITIONS OF ANY KIND, either express or implied. See the License for the
 // specific language governing permissions and limitations under the License.
 
-#if !defined(_WIN32)
+#if !defined(__SCE__)
```

Comment:
Currently, the way we pick up source files is that we choose the main folder
(which picks up this file), and then the platform folder (which will pick up e.g. `win32/file_descriptor_win32.cpp`)